### ISSUE TITLE
UX: Overlay elements like the date picker should prevent clicks on background elements

### ISSUE THREAD

**Issue by labkode** [owncloud/phoenix#3917]

**Comment by PVince81** [comment]
@LukasHirt will the new drop element we talked about yesterday help with this ? https://github.com/owncloud/owncloud-design-system/pull/816

**Comment by LukasHirt** [comment]
> will the new drop element we talked about yesterday help with this ?

Not really. The drop element still allows you to click on a different element. From what I understand this is more about modals though. Our latest modal component already prevents clicking on anything which is not part of that modal. The date picker is coming from a library though so we need to see if it's actually possible for us to fix this (personally am hoping we could drop this modal approach for date picker).

**Comment by pascalwengerter** [comment]
Closing as resolved, we've switched datepicker components quite a while ago and I can't remember anyone complaining about it receiving clicks that affect UI components behind it
